todo_app/data/session_items.py

```diff
@@ -53,6 +54,17 @@ def add_item(title):
 
     return item
 
+def remove_items_by_id(ids):
+    for id in ids:
+        currentItems = session['items'] 
+        lambda item: print(item['id'] != id), currentItems
+        session['items'] = [item for item in currentItems if item.get('id') != int(id)]
+
+def mark_items_as_completed(ids):
+    for id in ids:
+        item = get_item(id)
+        item['status'] = 'Completed'
```

Comment:
Minor, but it might be slightly nicer to use enums for item status rather than strings